test/functions/iterators/recursive/recursive-iter-findfilesfailure.good

```diff
@@ -0,0 +1,5 @@
+recursive-iter-findfilesfailure.chpl:29: In iterator 'mywalkdirs':
+recursive-iter-findfilesfailure.chpl:39: warning: module-level symbol is hiding function argument 'sort'
+$CHPL_HOME/modules/packages/Sort.chpl:466: note: found 'sort' defined here
+$CHPL_HOME/modules/packages/Sort.chpl:466: note: found 'sort' defined here
```

Comment:
Why does this version of 'sort' get printed twice?  My best guess was "maybe we created two instantiations for it?", but that seems unlikely given how early in compilation we are when printing this I think?

Reply:
This is a bug to be fixed by https://github.com/chapel-lang/chapel/pull/21711.